index.html

```diff
@@ -475,64 +475,67 @@
 ## User Agents {#user-agents}
 
 The <dfn>user agent</dfn> acts as an intermediary between a [=user=] and the web. The
-[=user agent=] is <em>not</em> a [=context=] in that it is expected to coincide with the
-[=subject=] and operate exclusively in the [=subject=]'s interest. It is <em>not</em> the
-[=first party=]. The [=user agent=] serves the [=user=] in a
-<dfn data-lt="fiduciary relationship">relationship of fiduciary agency</dfn>: it always puts
-the [=user=]'s interest first, up to and including, on occasion, protecting the [=user=]
-from themselves by preventing them from carrying out a harmful decision, or at the very
-least by speed-bumping it [[?FIDUCIARY-UA]]. For example, the [=user agent=] will make it
-difficult for the [=user=] to connect to a site the authenticity of which is hard to
-ascertain, will double-check that the user really intends to expose a sensitive device
-capability, or will prevent the [=user=] from consenting to permanent monitoring of their
-behaviour. Its <dfn>fiduciary duties</dfn> include [[?TAKING-TRUST-SERIOUSLY]]:
+[=user agent=] is <em>not</em> a [=context=] in that it is expected to align fully with its
+[=user=] and operate exclusively in that [=person=]'s interest. It is <em>not</em> the
+[=first party=]. The [=user agent=] serves the [=user=] as a <dfn>trustworthy agent</dfn>:
+it always puts the [=user=]'s interest first. In some occasions, this can mean protecting
+the [=user=] from themselves by preventing them from carrying out a dangerous decision,
+or at the very least by slowing down the user in their decision. For example, the
+[=user agent=] will make it difficult for the [=user=] to connect to a site if it can't verify
+that the site is authentic. It will check that the [=user=] really intends to expose a
+sensitive device to a page. It will prevent the [=user=] from consenting to the permanent
+monitoring of their behaviour. Its <dfn>user agent duties</dfn> include
```

Comment:
Related to #62, the UA won't know about all of the user's consents or all the monitoring around them, but we could probably be more precise to get at something the UA can do here. No need to fix this in this PR.